CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Conversions
+- Added `float()` expression to force conversion to float values
+- Added `string()` expression to force conversion to string values
```

Comment:
I think these should be named toFloat() and toString(), as `float` and `string` seems not that unlikely to become keywords at some point.... It also communicates a bit better what they do IMO. Or are you thinking about these like constructors?

Reply:
It was the syntax duckers first propose to me. I think it makes sense as we won't likely introduce actual object construction. But in that sense, these can be well interpreted as conversion constructors.

There should never be a symbol overlap in this scope. It's controlled what is export to it -- names don't leak around.

Reply:
Well, my point was that if "float" and "string" becomes keywords, this probably won't compile any more, and if it would, these functions would not be reached through them. Yeah, I'm aware that such a change would already be backwards-incompatible, but I don't really want us to paint ourselves further into a corner.

That being said, I don't really feel strongly about this. It was just my first reaction, that I expected these to be called something else.

Reply:
I think the idea was that these are essentially keywords. The just don't need to be programmed at the UX compiler level and work fine as functions.  That is, any introduction of a keyword would reasonably have the same behaviour, so I think we'd be fine. Not that I see any reason for those eventual keywords (this is just an expression language, not a full programming language).

Let me ask in #ux first. It's no big deal to change the name now.

Reply:
I think you're right that introduction of this as keywords would be reasonable to expect to do the same, so my objection is probably moot.

Reply:
yeah I agree with @mortoray here; if we want `float` and `string` as fn's it essentially reserves the symbols anyways in the same way keywords would be, and if we ever did anything with the same symbols later we'd want to play nice with these kinds of features anyways. So I'm on board 👍